react/components/LogoutCustomerSession.tsx

```diff
@@ -49,19 +53,39 @@ interface Props {
 const LogoutCustomerSession = (props: Props) => {
   const { client, loading, onDepersonify, attendantEmail, mobile } = props
   const handles = useCssHandles(CSS_HANDLES)
+  const { orderForm } = useOrderForm()
 
-  const getClientName = (client: any) =>
-    !!client
-      ? client.name.includes('null')
-        ? client.email.slice(0, client.email.indexOf('@'))
-        : client.name
+  const { id } = orderForm
+
+  const [customerClass, setCustomerClass] = useState('')
+
+  useEffect(()=> {
+    function getCustomerClass() {
+      const requestOptions = {
+        method: 'GET',
+      }
+      return fetch(`/api/checkout/pub/orderForm/${id}`, requestOptions)
+    }
+  
+    getCustomerClass().then(response => response.json()).then(data => {
+      if(!!data?.clientProfileData?.customerClass) {
+        setCustomerClass(data.clientProfileData.customerClass)
+      }
+    })
+  }, [])
+
+  const getClientName = (user: any) =>
+    !!user
+      ? user.name.includes('null')
+        ? user.email.slice(0, user.email.indexOf('@'))
+        : user.name
```

Comment:
Lint all changed files, please

Reply:
@igorbrasileiro LogoutCustomerSession.tsx is the only .tsx file in which I made the changes. I have already done linting for this file.

Reply:
Hey, can you change this ternary chain to if clauses?

Reply:
@igorbrasileiro Changed ternary chain to if clauses